mtp_noms_ops/apps/security/tasks.py

```diff
@@ -1,37 +1,41 @@
-from anymail.message import AnymailMessage
 from django.conf import settings
-from django.template import loader as template_loader
-from django.utils.translation import gettext, activate, get_language
+from django.template.defaultfilters import striptags
 from django.utils import timezone
+from django.utils.dateformat import format as format_date
+from django.utils.translation import gettext, activate, get_language
 from mtp_common.api import retrieve_all_pages_for_path
 from mtp_common.auth.api_client import get_api_session_with_session
 from mtp_common.spooling import spoolable
-from mtp_common.tasks import default_from_address, prepare_context
+from mtp_common.tasks import send_email
 from openpyxl.writer.excel import save_virtual_workbook
 
 from security.export import ObjectListSerialiser
 from security.utils import convert_date_fields
 
 
 @spoolable(body_params=('user', 'session', 'filters'))
-def email_export_xlsx(*, object_type, user, session, endpoint_path, filters, export_description,
-                      attachment_name):
+def email_export_xlsx(*, object_type, user, session, endpoint_path, filters, export_description):
     if not get_language():
         language = getattr(settings, 'LANGUAGE_CODE', 'en')
         activate(language)
 
     if object_type == 'credits':
-        export_message = gettext('Attached are the credits you exported from ‘%(service_name)s’.')
+        export_message = gettext('Click the link to download the credits you exported from ‘%(service_name)s’.')
```

Comment:
I wonder if we could get rid of the gettext altogether here and simplify given it would always be in English anyway and the rest of the email would be in English anyway (the rest of the template itself).

But I know you probably left this as it was instead of introducing even more changes.

Reply:
actually, i agree. i'll get rid of the gettext cos we don't want a mixed-language email to go out!